### DISCUSSION TITLE
How can I forced reset the dialog state of all from outside

### DISCUSSION THREAD

**Discussion by jekeam** [python-telegram-bot/python-telegram-bot#3334] [Q&A]
How can I reset the dialog state (Forced to start or .END point) of all users from an external script or using a bot command or Application()

Something like, but only so that the state is updated for the user himself:
```
for user in db_get_users():
    context.job_queue.application.persistence.conversations['main_dialog'][(user.id, user.id)] = 0
    await context.bot.send_message(chat_id=user.id, text='some text', reply_markup=get_main_markup())
```

**Comment by n0h4rt** [comment]
I want to know this too!

My problem is when I edit the `callback_data` of `InlineKeyboardButton` with `persistence` enabled. Any ongoing conversation can result in unwanted behavior.

Is there a way to do a soft reset instead of deleting persistent files?
